src/main/java/net/snowflake/client/core/StmtUtil.java

```diff
@@ -308,7 +307,7 @@ public static StmtOutput execute(StmtInput stmtInput) throws SFException, Snowfl
 
         String json = mapper.writeValueAsString(sqlJsonBody);
 
-        logger.debug("JSON: {}", (ArgSupplier) () -> SecretDetector.maskSecrets(json));
+        logger.debug("JSON: {}", json);
```

Comment:
Can you leave PR comments on every such instance as to why we're removing this detection?

Reply:
This is the same as above. 
SecretDetector.maskSecrets(json) causes the json to be passed through the secret detector here in this class but it is sent through again in the logger class after the arguments are formatted into the message string. https://github.com/snowflakedb/snowflake-jdbc/blob/master/src/main/java/net/snowflake/client/log/SLF4JLogger.java#L76 
https://github.com/snowflakedb/snowflake-jdbc/blob/master/src/main/java/net/snowflake/client/log/JDK14Logger.java#L136